<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ubuntu::Internal::CreateTargetIntroPage</class>
 <widget class="QWidget" name="Ubuntu::Internal::CreateTargetIntroPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>This Wizard will help to create build and runtime targets.&lt;br/&gt;
Please select for which type of device the target should be created for.</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>137</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="desktopButton">
        <property name="text">
         <string>Build and run your application locally on the Desktop.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="deviceButton">
        <property name="text">
         <string>Build  your application to run on a Ubuntu device or emulator.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="allImagesButton">
        <property name="text">
         <string>Show all available Images</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
